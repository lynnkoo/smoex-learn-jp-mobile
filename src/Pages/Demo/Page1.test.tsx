import React from 'react';
import renderer from 'react-test-renderer';
import Page1 from './Page1';

test('Page1 rendor correctly', () => {
  const tree = renderer.create(<Page1 />).toJSON();
  expect(tree).toMatchSnapshot();
});
