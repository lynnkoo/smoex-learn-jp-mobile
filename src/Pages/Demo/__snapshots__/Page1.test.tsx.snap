// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page1 rendor correctly 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 20,
          "margin": 10,
          "textAlign": "center",
        }
      }
    >
      Welcome to Page1
    </Text>
    <Text
      style={
        Object {
          "color": "#333333",
          "marginBottom": 5,
          "textAlign": "center",
        }
      }
    >
      Shake or press menu button for dev menu2
    </Text>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#333333",
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 20,
            "margin": 10,
            "textAlign": "center",
          }
        }
      >
        Go to List
      </Text>
    </View>
  </View>
</View>
`;
